<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="codeButton">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;可在此编辑文本内容&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QPushButton" name="selectFileButton">
            <property name="text">
             <string>选择文本文件</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="saveFileButton">
            <property name="text">
             <string>保存文本文件</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="versionLabel">
            <property name="text">
             <string>版本:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="versionSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="SymbolNumLabel">
            <property name="text">
             <string>码元数:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLCDNumber" name="symbolNumLcdNumber"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="generateButton">
          <property name="text">
           <string>生成</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportButton">
          <property name="text">
           <string>导出QR Code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportAsButton">
          <property name="text">
           <string>导出QR Code为</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="statisticsMenu">
    <property name="title">
     <string>统计信息</string>
    </property>
    <addaction name="showStatisticsAction"/>
    <addaction name="clearLogAction"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>解码</string>
    </property>
    <addaction name="openFileForDecodeAction"/>
   </widget>
   <addaction name="statisticsMenu"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="encodeSuccessRateAction">
   <property name="text">
    <string>&amp;success rate</string>
   </property>
  </action>
  <action name="encodeBytesCountAction">
   <property name="text">
    <string>&amp;bytes count</string>
   </property>
  </action>
  <action name="encodeTimesAction">
   <property name="text">
    <string>&amp;encode times</string>
   </property>
  </action>
  <action name="dncodeSuccessRateAction">
   <property name="text">
    <string>&amp;success rate</string>
   </property>
  </action>
  <action name="encodeBytesCountAction_2">
   <property name="text">
    <string>&amp;bytes count</string>
   </property>
  </action>
  <action name="decodeTimesAction">
   <property name="text">
    <string>&amp;decode times</string>
   </property>
  </action>
  <action name="openFileForDecodeAction">
   <property name="text">
    <string>从文件中读取二维码</string>
   </property>
   <property name="toolTip">
    <string>从文件中读取二维码</string>
   </property>
  </action>
  <action name="showStatisticsAction">
   <property name="text">
    <string>结果统计</string>
   </property>
  </action>
  <action name="clearLogAction">
   <property name="text">
    <string>清空历史</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>textEdit</tabstop>
  <tabstop>generateButton</tabstop>
  <tabstop>selectFileButton</tabstop>
  <tabstop>saveFileButton</tabstop>
  <tabstop>exportAsButton</tabstop>
  <tabstop>exportButton</tabstop>
  <tabstop>versionSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
